### ISSUE TITLE
FG not working as intended with HDR in Cyberpunk

### ISSUE THREAD

**Issue by Alan-Horta** [Nukem9/dlssg-to-fsr3#377]
I have tried DLSS to FSR3 in cyberpunk 2077 and looked like garbage.
But after seeing the DF video about the mode i tried it again and discovered that if HDR is disabled the mod works as intended.
I used my phone camera to record in slow motion. And apparently when HDR is activated it just duplicates frames.

I believe this also has to do with issue #300 
The log doesnt report anything

**Comment by ForthtuN** [comment]
I don't know what you mean about it looking like garbage, it looks the same to me, no distortion in colors or anything.

However, applying any vibrance it seems (or some other effect) will make Fsr3 behave really weird and just add input delay and duplicated frames (says 150 fps but it is like 50 fps). This behaviour is also if you open the Nvidia Overlay while in game and add effects, like adding more vibrance through color filter. It will instantly start to perform like shit, even after turning it off. Its also enough to just open the overlay and closing it and it will loose performance it seems, making fsr3 act weirdly. Even after disabling the filter it will still be bad until you restart the game by loading it again or exit to main menu. 

This is also in for example Warzone using both official FSR3 and this Fsr3 mod. It will behave very bad and loosing alot of performance using any filter added through nvidia game filter.

**Comment by TimmyP777** [comment]
Yeah I get this too. Saturation in "HDR settings" to zero and its butter smooth.

New drivers, game patches, etc can possibly set this back to .3. Did in my case.
